src/coreclr/src/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Windows.cs

```diff
@@ -10,5 +10,18 @@ private static bool GetGlobalizationInvariantMode()
         {
             return GetInvariantSwitchValue();
         }
+
+        private static bool GetUseIcuMode()
+        {
+            if (!Invariant)
+            {
+                if (GetUseNlsSwitchValue() || Interop.Globalization.LoadICU() == 0)
+                {
+                    return false;
+                }
+            }
+
+            return true;
```

Comment:
This will return true for InvariantMode. I would expect it to return false.

(It should not matter, but it is confusing to UseIcu to be true when we are not using ICU.)

Reply:
Yeah that's true, I just used `true` as an arbitrary value as it shouldn't matter, but you made a good point.